internal/datastore/mysql/datastore.go

```diff
@@ -669,3 +672,9 @@ func buildLivingObjectFilterForRevision(revision datastore.Revision) queryFilter
 func currentlyLivingObjects(original sq.SelectBuilder) sq.SelectBuilder {
 	return original.Where(sq.Eq{colDeletedTxn: liveDeletedTxnID})
 }
+
+type debugLogger struct{}
+
+func (debugLogger) Print(v ...any) {
+	log.Logger.Debug().CallerSkipFrame(1).Msg(fmt.Sprint(v...))
+}
```

Comment:
Is this specific to MySQL? Is it worth extending to the other datastores as well? It's also okay if that's outside of the scope of this PR.

Reply:
every datastore seems to have its own interface -- this particular one is specific to mysql, alas....